astroML/stats/tests/test_stats.py

```diff
@@ -133,8 +133,60 @@ def check_fit_bivariate_normal(sigma1, sigma2, mu, alpha, N=1000):
     assert_allclose(sigma1_fit, sigma1, rtol=rtol)
     assert_allclose(sigma2_fit, sigma2, rtol=rtol)
 
+
 def test_fit_bivariate_normal(sigma1=2.0, sigma2=1.0, N=1000):
     np.random.seed(0)
     mu = [10, 10]
     for alpha in np.linspace(-np.pi / 2, np.pi / 2, 7):
         yield check_fit_bivariate_normal, sigma1, sigma2, mu, alpha, N
+
+
+#------------------------------------------------------
+# Check truncated exponential and linear functions
+def test_trunc_exp():
+    x = np.linspace(0, 10, 100)
+    k = 0.25
+    xlim = [3, 5]
+    trunc_exp = trunc_exp_gen(name="trunc_exp", shapes='a, b, k')
```

Comment:
Instead of testing this explicitly, should probably just do ``from astroML.stats.random import trunc_exp``. The generator class is meant to be a private method, not a user-facing method. (we should probably add an ``__all__`` indicator to ``random.py`` to make this more clear)